packages/worker/README.md

```diff
@@ -0,0 +1,7 @@
+# @bugsnag/worker
+
+This package contains the worker implementation of the Bugsnag notifier for JavaScript. The normal use case is to install this package via `@bugsnag/js`, but you can install it directly if you want to.
+
+## License
+
+This package is free software released under the MIT License. See [LICENSE.txt](./LICENSE.txt) for details.
```

Comment:
This file is missing — it's the same license in every package so you can just copy paste it